src/util/chunked_content_decoder.js

```diff
@@ -71,7 +78,7 @@ class ChunkedContentDecoder extends stream.Transform {
                 this.chunk_size -= content.length;
                 index += content.length - 1;
                 if (content.length) this.push(content);
-                if (!this.chunk_size) this.state = STATE_WAIT_CR_DATA;
+                if (!this.chunk_size) this.state = STATE_WAIT_TRAILER_DATA;
```

Comment:
we need to switch to trailer state only on last chunk.
```suggestion
                if (!this.chunk_size) {
                    this.state = this.last_chunk ? STATE_WAIT_TRAILER_DATA : STATE_WAIT_CR_DATA;
                }
```